.github/workflows/test.yaml

```diff
@@ -6,7 +6,7 @@ jobs:
     strategy:
       matrix:
         go-version: [1.19.x]
-        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
I dropped this because we are not releasing the manager for mac and windows, not sure if there are other reasons to keep them around